web/impact/impact/tests/test_utils.py

```diff
@@ -1,22 +1,28 @@
 # MIT License
 # Copyright (c) 2017 MassChallenge, Inc.
 
-from test_plus.test import TestCase
-from impact.utils import get_profile
-from impact.tests.contexts import (
-    UserContext,
-)
-from impact.models import (
-    ExpertProfile,
-    MemberProfile,
+from impact.models.utils import (
+    model_name_to_snake,
+    snake_to_model_name,
 )
+from impact.tests.api_test_case import APITestCase
+
+
+empty_string = ''
+model_name = 'ProgramStartupStatus'
+lowercase_model_name = 'programstartupstatus'
+snake_case = 'program_startup_status'
+
+
+class TestUtils(APITestCase):
+    def test_empty_string_snake_case_returns_nothing(self):
+        assert model_name_to_snake(empty_string) == ''
 
+    def test_model_name_to_snake_case(self):
+        assert model_name_to_snake(model_name) == snake_case
 
-class TestUtils(TestCase):
-    def test_get_profile_for_expert(self):
```

Comment:
Why is this test deleted?

Reply:
@yotammanor Did you make these changes?  I know you merged things yesterday, and I created this file from scratch.  I have no idea what this code is.